aws/table_aws_ec2_instance.go

```diff
@@ -609,3 +618,15 @@ func getEc2InstanceTurbotTitle(_ context.Context, d *transform.TransformData) (i
 	}
 	return title, nil
 }
+
+func ec2InstanceStateChangeTime(_ context.Context, d *transform.TransformData) (interface{}, error) {
+	data := d.HydrateItem.(*ec2.Instance)
+	if helpers.StringSliceContains([]string{"shutting-down", "stopped", "stopping", "terminated"}, *data.State.Name) {
```

Comment:
What happens if the state isn't in this list, i.e., starting an instance? Are there other states we need to handle?

Reply:
@cbruno10 For the other states, `state_change_time` column value will be `null`.
For the above mention states, AWS gives the state change time in `StateTransitionReason`, while for the other states, `StateTransitionReason` returns `null`.

Reply:
@Subhajit97 Thanks, it is possible to use the launch time here if the instance is `running`, since that time changes every time a stopped instance is started?

Also, for the column name, should it be `state_transition_time` instead, as this seems more consistent with other properties the instance has, e.g., `StateTransitionReason`?

Reply:
sure @cody, thanks! I will update the function.